### DISCUSSION TITLE
API key is not showing

### DISCUSSION THREAD

**Discussion by Aliinoccent** [Zain-ul-din/LGU-BetterTimeTable#35] [Q&A] [answered]
An API key is not showing on your website even if I'm logged in

**Comment by Zain-ul-din** [comment, marked as answer]
@Aliinoccent we are no longer providing that APIS since you  can only make limited calls to them. Now if you wanna get data go to [open_db](https://github.com/Zain-ul-din/lgu-crawler/tree/master/db) and check [api_usage](https://github.com/Zain-ul-din/lgu-crawler/tree/master/db) for documentation.
